tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -340,6 +340,41 @@ def make_csv_dataset_v2(
   tuple that corresponds to a batch of CSV rows. The features dictionary
   maps feature column names to `Tensor`s containing the corresponding
   feature data, and labels is a `Tensor` containing the batch's label data.
+  
+  Usage Example:
+  
+  Using IRIS dataset to show how to convert .csv file into a dataset.
+  
+  >>> train_dataset_url = "https://storage.googleapis.com/download.tensorflow.org/data/iris_training.csv"
+  >>> train_dataset_fp = tf.keras.utils.get_file(fname=os.path.basename(train_dataset_url), origin=train_dataset_url)
+  >>>
+  >>> column_names = ['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'species']
+  >>> feature_names = column_names[:-1]
+  >>> label_name = column_names[-1]
+  >>> 
+  >>> batch_size = 32
+  >>> train_dataset = tf.data.experimental.make_csv_dataset(
+  ... train_dataset_fp,
+  ... batch_size,
+  ... column_names=column_names,
+  ... label_name=label_name,
+  ... num_epochs=1)
+  >>> features_batch, labels_batch = next(iter(train_dataset))
+  >>> print(features_batch)
+  <OrderedDict([('sepal_length', <tf.Tensor: id=138, shape=(32,), dtype=float32, numpy=
+array([4.9, 5.1, 5.4, 6.8, 6.6, 6.7, 5. , 6.2, 5.1, 6.2, 6. , 6.4, 6.2,
```

Comment:
This is supposed to be aligned similar to the line above. Add necessary whitespace for this and all similar lines.